nodejs/week1/avg.js

```diff
@@ -0,0 +1,25 @@
+
+'use strict';
+/* const comand_arguments = process.argv
+function findAverage(a, b, c) {
+    // To extract number from string
+    return Math.round(parseInt(a)+parseInt(b)+parseInt(c) / 3);
+}
+const average = findAverage(comand_arguments[2], comand_arguments[3], comand_arguments[4] )
+console.log(average); */
+
+const comand_arguments = process.argv
+console.log('comand arguments', comand_arguments)
+function findAverage(numbers) {
+    let sum = 0;
+    for (let index = 2; index < numbers.length; index++) {
+        console.log('index is', numbers[index])
+        sum += parseInt(numbers[index]);
+        console.log('sum in loop is', sum)
+    }
+    console.log('sum is', sum)
+    return Math.round(parseInt(sum / numbers.length))
+}
+const average = findAverage(comand_arguments)
```

Comment:
we might want to check if the user has provided any numbers after node avg.js, and log an error if not